USAGE

```diff
@@ -0,0 +1,54 @@
+toTemplate_flow pipeline
+=====================
+
+Register t1, metrics and tractograms to template space.
+We suggest to use https://github.com/scilus/combine_flows to create input.
+
+
+USAGE
+
+nextflow run main.nf [OPTIONAL_ARGUMENTS] --input --template
+
+DESCRIPTION
+
+    --input=/path/to/[root]     Root folder containing multiple subjects
+                                                [root]
+                                                ├── S1
+                                                │   │── *t1.nii.gz (diff space)
+                                                |   └── metrics (optional)
+                                                |   │   └── *.nii.gz
+                                                |   └── tractograms (optional)
+                                                |       └── *.trks
+                                                └── S2
+                                                    └── *
+
+    --template                                  Template used for registration.
+
+OPTIONAL ARGUMENTS (current value)
+    --linear                                    If set, it will use linear registration instead of non linear registration.
+
+    --slow_registration                         If set, it will use antsRegistrationSyN instead of antsRegistrationSyNQuick for registration.
+
+    --output_dir                                Specify a new output directory to avoid overwriting results (when dealing with multiple input labels) ($output_dir)
+
+    --processes_register                        Number of processes for registration task ($processes_register)
+
+AVAILABLE PROFILES (using -profile option (e.g. -profile use_cuda,fully_reproducible))
```

Comment:
Is `use_cuda` an available profile ? I get that it is an example but maybe we should put an available profile for example?